app/templates/includes/_footer.html

```diff
@@ -0,0 +1,44 @@
+<footer class="footer container-fluid">
```

Comment:
If it's not too much work, might be better to move/rename these files rather than copy. That way Git will show the whole history rather than treating this as brand new